resources/config/services/console.yml

```diff
@@ -5,17 +5,14 @@ services:
 
   OpenCFP\Console\Command\:
     resource: '%kernel.project_dir%/classes/Console/Command/*'
-
-  OpenCFP\Console\Command\ClearCacheCommand:
-    arguments:
-      $paths: ['%kernel.cache_dir%']
+    tags:
+      - { name: console.command }
```

Comment:
@chartjes @derrabus @mdwheele 

I think with this it should be safe going forward and having the commands automatically registered - at least https://symfony.com/doc/2.6/cookbook/console/commands_as_services.html suggests to do so.

Running

```
$ bin/console
```

yields

```
./bin/console
Symfony 3.4.2 (kernel: OpenCFP, env: development, debug: true)

Usage:
  command [options] [arguments]

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -e, --env=ENV         The environment name [default: "development"]
      --no-debug        Switches off debug mode
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Available commands:
  about                      Displays information about the current project
  help                       Displays help for a command
  list                       Lists commands
 assets
  assets:install             Installs bundles web assets under a public directory
 cache
  cache:clear                Clears the cache
  cache:pool:clear           Clears cache pools
  cache:pool:prune           Prune cache pools
  cache:warmup               Warms up an empty cache
 config
  config:dump-reference      Dumps the default configuration for an extension
 debug
  debug:autowiring           Lists classes/interfaces you can use for autowiring
  debug:config               Dumps the current configuration for an extension
  debug:container            Displays current services for an application
  debug:event-dispatcher     Displays configured listeners for an application
  debug:form                 Displays form type information
  debug:router               Displays current routes for an application
  debug:swiftmailer          Displays current mailers for an application
  debug:translation          Displays translation messages information
  debug:twig                 Shows a list of twig functions, filters, globals and tests
 eloquent
  eloquent:make:seeder       Create a new seeder class
  eloquent:migrate           Executes a migration.
  eloquent:migrate:fresh     Drop all tables and re-run all migrations.
  eloquent:migrate:install   Creates the migration repository.
  eloquent:migrate:make      Creates a new migration file
  eloquent:migrate:refresh   Reset and re-run all migrations
  eloquent:migrate:reset     Rollback all database migrations
  eloquent:migrate:rollback  Rollback the last database migration
  eloquent:migrate:status    Show the status of each migration
  eloquent:seed              Seed the database with records
 lint
  lint:twig                  Lints a template and outputs encountered errors
  lint:xliff                 Lints a XLIFF file and outputs encountered errors
  lint:yaml                  Lints a file and outputs encountered errors
 router
  router:match               Helps debug routes by simulating a path info match
 swiftmailer
  swiftmailer:email:send     Send simple email message
  swiftmailer:spool:send     Sends emails from the spool
 translation
  translation:update         Updates the translation file
 user
  user:create                Creates a new user
  user:demote                Demote an existing user from a role
  user:promote               Promote an existing user to a role
```

💡 Note our commands at the bottom.